### ISSUE TITLE
关于掩码生成问题

### ISSUE THREAD

**Issue by Dante-7777** [decisionintelligence/DUET#12]
您好，有些问题想和您探讨一下
resample_matrix = gumbel_softmax(logits,hard=True)
在gumbel采样时，我尝试修改默认参数tau的值
resample_matrix = gumbel_softmax(logits,tau=0.1, hard=True)
发现几次修改后loss值几乎毫无变化，原因是什么？
其次我在自己的模型中使用全1掩码矩阵和本论文中提到的掩码矩阵，最终的结果也几乎毫无变化，这个真的可以学习到多变量之间的关系？
